src/ansys/fluent/core/utils/fluent_version.py

```diff
@@ -74,6 +74,7 @@ class FluentVersion(Enum):
     FluentVersion.v232.awp_var == 'AWP_ROOT232'
     """
 
+    v261 = "26.1.0"
```

Comment:
To get PyFLuent working with local dev Fluent

Reply:
I already have this as a PR. Will merge soon.